packages/modules/sentry/src/edge-api-sentry-logger.ts

```diff
@@ -1,19 +1,23 @@
-import { createGetModuleFromFilename } from './module'
-
-import { createEventEnvelope, getEnvelopeEndpointWithUrlEncodedAuth } from '@sentry/core'
+import { SeverityLevel } from '@sentry/types'
 import {
   stackParserFromStackParserOptions,
   createStackParser,
   nodeStackLineParser,
   exceptionFromError,
-  dsnFromString,
 } from '@sentry/utils'
+import {
+  RCQuery,
```

Comment:
Should we log a ticket to rename these types in edge-api-sdk so they aren't RC specific?

Reply:
good idea